File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/response/PantallaInmueblesResponseDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.inhometfgcarloshernandez.spring.model.CajonDTO;
import org.tfg.inhometfgcarloshernandez.spring.model.MuebleDTO;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PantallaInmueblesResponseDTO {
    private List<String> habitaciones;
    private List<MuebleDTO> muebles;
    private List<CajonDTO> cajones;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/response/PantallaInmueblesResponseDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model.response

import com.example.inhometfgandroidcarloshernandez.data.model.CajonDTO
import com.example.inhometfgandroidcarloshernandez.data.model.HabitacionDTO
import com.example.inhometfgandroidcarloshernandez.data.model.MuebleDTO

data class PantallaInmueblesResponseDTO (
    val habitaciones: List<HabitacionDTO> = emptyList(),
    val muebles: List<MuebleDTO> = emptyList(),
    val cajones: List<CajonDTO> = emptyList()
)